compute-budget/src/compute_budget.rs

```diff
@@ -180,7 +182,8 @@ impl ComputeBudget {
             alt_bn128_multiplication_cost: 3_840,
             alt_bn128_pairing_one_pair_cost_first: 36_364,
             alt_bn128_pairing_one_pair_cost_other: 12_121,
-            big_modular_exponentiation_cost: 33,
+            big_modular_exponentiation_base_cost: 190,
+            big_modular_exponentiation_cost_multiplicative_factor: 2,
```

Comment:
Isn't it a "divisor" or "denominator" instead of a `multiplicative_factor`?

Reply:
Yeah you are right. 1/2 is the multiplicative factor. I'll rename it to divisor.